contract_discount/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Contract Discount
```

Comment:
Put the same line in the top for highlighting the title (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)